extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -3,7 +3,6 @@
 const async = require('async');
 const { errors, versioning } = require('arsenal');
 const { ObjectMD } = require('arsenal').models;
```

Comment:
we need to create a ticket to remove this from Arsenal, eventually
(i.e. after we have migrated all branches to this new backbeat)

Reply:
=> https://scality.atlassian.net/browse/ARSN-435